azerothcore/azerothcore-wotlk
# feat(Core/BG): track who accept to join BG

Why this? because I would track who joins BGs.
**Scenario**: the same person joins BG with 2 characters and with one of the two do not enter to BG it makes an unbalanced game inside the BG.

Actually, we track only the char that does not enter, but even who enter can be considered an abuser because in this scenario he is the same person with same IP probably.

I will update the tool [BG-Queue-Abuser-Viewer](https://github.com/Helias/BG-Queue-Abuser-Viewer) for this.

## Changes Proposed:
-  track the players who accepts to join BG and not only who leave it

## Tests Performed:
- tested in-game

## How to Test the Changes:
- .debug bg
- join BG
- enter to BG and check battleground_deserters table, there will be a type 5 that stands for BG accepted

## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
